CMakeLists.txt

```diff
@@ -76,6 +76,10 @@ else()
 			OUTPUT_STRIP_TRAILING_WHITESPACE)
 endif()
 
+if(WIN32)
```

Comment:
Hm, only GNU/linux is supported I suppose.
Anyways, anyone trying to build on other system is going to be stuck on libpmemobj not found, and libsyscall_intercept not found, so whatever.